bundle/manifests/activemq-artemis-operator.clusterserviceversion.yaml

```diff
@@ -2570,7 +2570,7 @@ spec:
             spec:
               containers:
               - args:
-                - --zap-log-level=info
+                - --zap-log-level=error
```

Comment:
@gaohoward @gtully we have already discussed this offline but I still think we should use info level by default. If it is too  much verbose we could move some log messages to debug level `logger.V(1).Info(...`

Reply:
Yeah It would be more conventional to set info as default level. This also make it easy to print the start up logs in normal info format without having to use a separate logger (the setup logger). This is because the error log doesn't print simple message lines, it also print the stack trace even a nil err is passed in, polluting the log output.

Reply:
I'll change it back to info if no objection. @brusdev @gtully